pkg/yurthub/otaupdate/ota.go

```diff
@@ -87,29 +97,48 @@ func UpdatePod(clientset kubernetes.Interface, nodeName string) http.Handler {
 		namespace := params["ns"]
 		podName := params["podname"]
 
-		err, ok := applyUpdate(clientset, namespace, podName, nodeName)
-		// Pod update failed with error
-		if err != nil {
-			WriteErr(w, "Apply update failed", http.StatusInternalServerError)
-			return
-		}
+		pod, ok := preCheck(clientset, namespace, podName, nodeName)
```

Comment:
It seems that preCheck is not suitable for static pod upgrade. we need to add some verification for avoiding static pod upgrading reenter.

Reply:
The new static pod won't have `PodNeedUpgrade` condition and it's not likely to re-enter the upgrade process.